src/main/kotlin/blackJack/domain/player/Participant.kt

```diff
@@ -4,12 +4,14 @@ import blackJack.domain.card.Card
 import blackJack.domain.card.Cards
 import blackJack.domain.enums.Status
 
-open class Participant(var status: Status, var cards: Cards) {
+open class Participant(var status: Status = Status.HIT, var cards: Cards = Cards(emptyList())) {
```

Comment:
https://github.com/next-step/kotlin-blackjack/pull/678#discussion_r1410302988 코멘트 반영 잘 해주셨어요!

소소한 피드백 드리자면, var status 가변적인 부분은 외부에 노출안해도 될 것 같아요. 추가로. cards도 Cards 클래스 내부에서 변경이 가능한 부분이라 val로 해주셔도 될 것 같아요 😊

```suggestion
open class Participant {
    var status: Status = Status.HIT
        private set
    val cards: Cards = Cards(emptyList())
```

Reply:
위 피드백 대로 반영 하였고 그에 따라 testCode 도 수정하였습니다😊😊